src/tools/dart-analyze.md

```diff
@@ -0,0 +1,44 @@
+---
+title: dart analyze
+description: Command-line tool for static analysis
+toc: false
+---
+
+The `dart analyze` command (previously called `dartanalyzer`)
+performs the same [static analysis][]
+that you get when you use an IDE or editor that has Dart support.
+
+Here's an example of performing static analysis over all the Dart files
+under the current directory:
+
+```terminal
+$ dart analyze
+```
+
+You can add a directory argument:
+
+```terminal
+$ dart analyze bin
+```
+
```

Comment:
Might be worthwhile to mention the `--fatal-infos` flag, which is commonly used in CI scripts (see e.g. https://github.com/dart-lang/.github/blob/master/workflow-templates/test-package.yaml#L36)